pkgs/by-name/zo/zoom-us/package.nix

```diff
@@ -6,19 +6,49 @@
   xdgDesktopPortalSupport ? true,
   callPackage,
   buildFHSEnv,
+  gzip,
```

Comment:
I think `gzip` is already in stdenv, so we don't need to list it here (see also below)